libs/ledger-live-common/src/exchange/swap/getExchangeRates.ts

```diff
@@ -40,22 +41,62 @@ const getExchangeRates: GetExchangeRates = async (
   const amountFrom = transaction.amount;
   const tenPowMagnitude = new BigNumber(10).pow(unitFrom.magnitude);
   const apiAmount = new BigNumber(amountFrom).div(tenPowMagnitude);
+
+  const providerList = providers
+    .filter((item) => {
+      const index = item.pairs.findIndex(
+        (pair) =>
+          pair.from === "ethereum/erc20/btrfly" &&
+          pair.to === "ethereum/erc20/reflex"
```

Comment:
should be `exchange.from` / `exchange.to` ?

Reply:
Yes, sorry wrong extra change.